NeoML/include/NeoML/Dnn/DnnBlob.h

```diff
@@ -200,10 +200,14 @@ class NEOML_API CDnnBlob : public IObject {
 protected:
 	virtual ~CDnnBlob();
 
+	CDnnBlob( IMathEngine& _mathEngine, const CBlobDesc& _desc, CMemoryHandle _data ) :
+		mathEngine( _mathEngine ), desc( _desc ), data( _data ), dataOwned( false ), parentPos( 0 ) {}
```

Comment:
_Discussabe_
1) It's better to add `NeoAssert( desc.DataType != CT_Invalid );` otherwise it's possible to serialize into this blob after constructor (and this loading won't reset `dataOwned` flag and that will be a leak)
2) What about storing such blobs? Do we wanna allow that scenario?

Or, at least, we can completely disable any serialization by `NeoAssert`.

Reply:
1. Fixed.
2. It works. We can't load such blobs but can store them (the second NeoAssert guarantees it).